articles/dms/tutorial-mysql-azure-mysql-offline-portal.md

```diff
@@ -0,0 +1,254 @@
+    
+    ![Azure Database Migration Service create](media/tutorial-mysql-to-azure-mysql-offline-portal/07-dms-portal-create-submit.png)
+
+## Create a migration project
+
+After the service is created, locate it within the Azure portal, open it, and then create a new migration project.  
+
+1. In the Azure portal, select **All services**, search for Azure Database Migration Service, and then select **Azure Database Migration Services**.
+
+    ![Locate all instances of Azure Database Migration Service](media/tutorial-mysql-to-azure-mysql-offline-portal/08-01-dms-portal-searchservice.png)
+
+2. Select your migration service instance from the search results and select + **New Migration Project**.
+    
+    ![Create a new migration project](media/tutorial-mysql-to-azure-mysql-offline-portal/08-02-dms-portal-newproject.png)
+
+3. On the **New migration project** screen, specify a name for the project, in the **Source server type** selection box, select **MySQL**, in the **Target server type** selection box, select **Azure Database For MySQL** and in the **Migration activity type** selection box, select **Data migration**
+
+    ![Create Database Migration Service Project](media/tutorial-mysql-to-azure-mysql-offline-portal/09-dms-portal-project-mysql-create.png)
+
+    > [!NOTE]
+    > Alternately, you can choose **Create project only** to create the migration project now and execute the migration later.
+
+## Configure migration project
+
+1. On the **Select source** screen, specify the connection details for the source MySQL instance, and select **Next : Select target>>**
+
+    ![Add source details screen](media/tutorial-mysql-to-azure-mysql-offline-portal/10-dms-portal-project-mysql-source.png)
+
+2. On the **Select target** screen, specify the connection details for the target Azure Database for MySQL instance, and select **Next : Select databases>>**
+
+    ![Add target details screen](media/tutorial-mysql-to-azure-mysql-offline-portal/11-dms-portal-project-mysql-target.png)
+
+3. On the **Select databases** screen, map the source and the target database for migration, and select **Next : Configure migration settings>>**
+    
+    If the target database contains the same database name as the source database, Azure Database Migration Service selects the target database by default.
+    ![Select database details screen](media/tutorial-mysql-to-azure-mysql-offline-portal/12-dms-portal-project-mysql-selectdb.png)
+    
+    > [!NOTE] 
+    > Though you can select multiple databases in this step, each instance of Azure Database Migration Service supports up to 4 databases for concurrent migration. Also, there is a limit of 10 instances of Azure Database Migration Service per subscription per region. For example, if you have 80 databases to migrate, you can migrate 40 of them to the same region concurrently, but only if you have created 10 instances of the Azure Database Migration Service.
```

Comment:
For offline there is no hard cap on the number of selected databases.  They can select as many as they want in a single migration task.  But there are limits to how many and how fast the DBs can be migrated this way, since each database will share compute.  Technically with the default configuration of the premium SKU, each migration task will attempt to migrate two tables in parallel.  These tables could be from any of the selected databases.  If this isn't fast enough, they can split database migration activities into different migration tasks and scale across multiple services.

Reply:
Done